### DISCUSSION TITLE
Project related issue

### DISCUSSION THREAD

**Discussion by PANDEYJI2710** [smartcontractkit/full-blockchain-solidity-course-py#1005] [Q&A] [answered]
Hello everyone I just got first project of my life but I do not know how to do it. If anyone help me in doing that project then I will share half of the prize of the project with him. 
Project can be made using solidity for ethereum blockchain instead of solana as well
[Blockchain Assignment.pdf](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/files/8038218/Blockchain.Assignment.pdf)

**Comment by cromewar** [comment, marked as answer]
Hello @PANDEYJI2710 
This section for the repo is for questions related to the course / theory if you want to get help on this project I strongly suggest you to visit chainlink Discord channel, you'll find a lot of developers there ;)
